.github/workflows/main.yml

```diff
@@ -23,3 +23,4 @@ jobs:
     uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
     with:
       plugin: foreman_theme_satellite
+      foreman_version: feat-36849-add-ruby-3-ga
```

Comment:
Note to self: we need to rebase this branch.

Reply:
or delete it and use Oleh's fork, which I did now :)